kind/common.sh

```diff
@@ -210,9 +211,15 @@ function _kind_up() {
 }
 
 function main() {
-    _kind_up
-
-    echo "cluster '$CLUSTER_NAME' is ready"
+    case $1 in
+    up)
```

Comment:
add a default action to `up` so the kepler-action can work as usual
https://github.com/sustainable-computing-io/kepler-action/blob/main/index.js#L39

@SamYuan1990

Reply:
I suppose we should have tag before we merge this.